README.md

```
@@ -297,6 +313,18 @@ Options:
   --remote TEXT       rclone remote name (e.g., 'nextcloud')
   --path TEXT         Remote path on Nextcloud (e.g., 'datasets/mydataset')
   --help              Show this message and exit.
+<<<<<<< HEAD
+  
+```
+#### Examples of using deploy command
+##### Mode 1: Classic Deploy (Distributions)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)